File: examples/quran/search/node/searchTheQuran.js
import * as quran from "@m3o/services/quran";

// Search the Quran for any form of query or questions
async function SearchTheQuran() {
  let quranService = new quran.QuranService(process.env.MICRO_API_TOKEN);
  let rsp = await quranService.search({
    query: "messenger",
  });
  console.log(rsp);
}

await SearchTheQuran();

File: examples/quran/search/go/searchTheQuran.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/quran"
)

// Search the Quran for any form of query or questions
func SearchTheQuran() {
	quranService := quran.NewQuranService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := quranService.Search(&quran.SearchRequest{
		Query: "messenger",
	})
	fmt.Println(rsp, err)
}
